VideoAPI/Video.API/Controllers/ConferenceController.cs

```diff
@@ -390,31 +393,46 @@ private async Task DeleteAudioRecordingApplication(Guid conferenceId)
 
             if (queriedConference != null && queriedConference.AudioRecordingRequired)
             {
-                await _audioPlatformService.DeleteAudioApplicationAsync(queriedConference.HearingRefId);
+                if (await CheckAudioRecordingFile(queriedConference.HearingRefId))
+                {
+                    await _audioPlatformService.DeleteAudioApplicationAsync(queriedConference.HearingRefId);
+                }
+                else
+                {
+                    var msg = $"Audio recording file not found for hearing: {queriedConference.HearingRefId}";
+                    var ex = new AudioPlatformException(msg, HttpStatusCode.NotFound);
```

Comment:
Same here, should not be just creating exceptions, just for logging.. if you know what I mean..

Reply:
What about AI?